lib/std/crypto/testdata/wycheproof_gen.zig

```diff
@@ -0,0 +1,93 @@
+// transforms https://raw.githubusercontent.com/C2SP/wycheproof/cd27d6419bedd83cbd24611ec54b6d4bfdb0cdca/testvectors/ecdsa_secp256r1_sha256_test.json into [_]TestVector{}
```

Comment:
I'm not sure the correct place to put scripts like this. All I know is someone must have written something similar for the current Project Wycheproof tests.

Reply:
There are a couple scripts used to build parts of the zig source code (thinking about the ones that create the header files, and the `libSystem` copies). They are maintained in distinct repositories.

Reply:
can also go in `tools/`

Reply:
I chose tools.